openssl/src/symm.rs

```diff
@@ -503,6 +539,11 @@ impl Crypter {
             Mode::Decrypt => CipherCtxRef::decrypt_init,
         };
 
+        if cfg!(ossl102) {
+            // Explicitly opt in the AES key wrap feature
+            ctx.set_flags(ffi::EVP_CIPHER_CTX_FLAG_WRAP_ALLOW);
```

Comment:
I don't think this should be done automatically.

Reply:
I updated to only set the flag if it's an aes wrap `cipher`. Does this work for you?

Reply:
I don't think this should be done automatically.

Reply:
Reverted the change in symm.